diff --git a/node_modules/scribe.js-ocr/js/generalWorkerMain.js b/node_modules/scribe.js-ocr/js/generalWorkerMain.js
index 57d1710..c81e817 100644
--- a/node_modules/scribe.js-ocr/js/generalWorkerMain.js
+++ b/node_modules/scribe.js-ocr/js/generalWorkerMain.js
@@ -352,7 +352,7 @@ export class gs {
    * @returns
    */
   static initTesseract = async ({
-    anyOk = true, vanillaMode = false, langs = ['eng'], config = {},
+    anyOk = true, vanillaMode = false, langs = ['eng'], config = {}, oem = null,
   }) => {
     await gs.schedulerReady;
 
@@ -369,12 +369,12 @@ export class gs {
     // data will be loaded over network from all workers (rather than downloading once and caching).
     const worker0 = gs.schedulerInner.workers[0];
     await worker0.reinitialize({
-      langs, vanillaMode, config, langPath: opt.langPath,
+      langs, oem, vanillaMode, config, langPath: opt.langPath,
     });
 
     if (gs.schedulerInner.workers.length > 0) {
       const resArr = gs.schedulerInner.workers.slice(1).map((x) => x.reinitialize({
-        langs, vanillaMode, config, langPath: opt.langPath,
+        langs, oem, vanillaMode, config, langPath: opt.langPath,
       }));
       await Promise.allSettled(resArr);
     }
diff --git a/node_modules/scribe.js-ocr/js/recognizeConvert.js b/node_modules/scribe.js-ocr/js/recognizeConvert.js
index f1f53e5..34b7426 100644
--- a/node_modules/scribe.js-ocr/js/recognizeConvert.js
+++ b/node_modules/scribe.js-ocr/js/recognizeConvert.js
@@ -582,7 +582,7 @@ async function recognizeAllPages(doc, legacy = true, lstm = true, mainData = fal
   }
 
   await gs.initTesseract({
-    anyOk: false, vanillaMode, langs, config,
+    anyOk: false, vanillaMode, langs, config, oem: legacy && lstm ? 2 : legacy ? 0 : 1,
   });
 
   // If Legacy and LSTM are both requested, LSTM completion is tracked by a second array of promises (`promisesB`).
